app.json

```diff
@@ -1,29 +1,46 @@
 {
   "name": "network-pulse-api",
-  "scripts": {
-  },
   "env": {
+    "HEROKU_APP_NAME": {
+      "required": true
+    },
+    "REDIRECT_URIS": {
+      "required": true
+    },
     "client_id": {
       "required": true
     },
     "client_secret": {
       "required": true
     },
-    "CORS_ORIGIN_WHITELIST": {
+    "AWS_ACCESS_KEY_ID": {
       "required": true
     },
-    "redirect_uris": {
+    "AWS_S3_CUSTOM_DOMAIN": {
       "required": true
-    }
-  },
-  "formation": {
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+      "required": true
+    },
+    "AWS_STORAGE_BUCKET_NAME": {
+      "required": true
+    },
+    "AWS_STORAGE_ROOT": {
+      "required": true
+    },
+    "USE_S3": "True",
+    "SSL_PROTECTION": "True",
+    "CORS_ORIGIN_ALLOW_ALL": "True"
   },
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql:hobby-dev"
   ],
   "buildpacks": [
     {
       "url": "heroku/python"
     }
-  ]
+  ],
+  "scripts": {
+    "postdeploy": "python manage.py review_app_admin"
```

Comment:
I didn't even know this was a thing we could do! awesome!

Reply:
Could you also add in a section in the README for review apps and how to locate the generated password?

Reply:
Yes, I need to add some docs :)